state/store.go

```diff
@@ -25,16 +26,37 @@ const (
 
 //------------------------------------------------------------------------
 
-func calcValidatorsKey(height int64) []byte {
-	return []byte(fmt.Sprintf("validatorsKey:%v", height))
+const (
+	// prefixes are unique across all tm db's
```

Comment:
We may want to gather them all in one place if we want to enforce uniqueness.

Reply:
That would make sense. Do we want to enforce uniqueness though? I thought this was more a minor thing to avoid negligent mistakes. Usually we have a separate db instance for each of the stores

Reply:
Might not be that important, as long as we make it clear that it's the caller's responsibility to use separate databases. That's probably a reasonable requirement, just call it out in the constructor.

Another option might be to use string prefixes instead, e.g. `blockmeta` -- this takes additional space though, but is more understandable and less likely to conflict. I'd go for numbers though.